src/Microsoft.Framework.Caching.SqlServer/SqlQueries.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.Framework.Caching.SqlServer
+{
+    internal class SqlQueries
+    {
+        private const string CreateTableFormat = "CREATE TABLE [{0}](" +
+            "Id nvarchar(100)  NOT NULL PRIMARY KEY, " +
```

Comment:
If we don't know how people are going to use it we could increase the size to 900. I think 900 sounds like a big number but I believe there is no downsize to use that.